src/platform_impl/linux/wayland/window/mod.rs

```diff
@@ -105,7 +106,7 @@ impl Window {
         let theme_manager = event_loop_window_target.theme_manager.clone();
         let mut window = event_loop_window_target
             .env
-            .create_window::<FallbackFrame, _>(
```

Comment:
I think it would make sense to keep both under the feature flag? Though, we should probably enable adwaita theme by default.

That's pretty straightforward with something like `type WinitFrame = FallbackFrame` and using appropriate type...